modules/users/tests/server/user.server.model.tests.js

```diff
@@ -57,59 +57,90 @@ describe('User Model Unit Tests:', function () {
     });
 
     it('should be able to save without problems', function (done) {
-      user.save(done);
+      var _user = new User(user);
+      _user.save(function (err) {
+        should.not.exist(err);
+        _user.remove(function (err) {
+          should.not.exist(err);
+          done();
+        });
+      });
     });
 
     it('should fail to save an existing user again', function (done) {
-      user.save(function () {
-        user2.save(function (err) {
+      var _user = new User(user);
+      var _user2 = new User(user2);
+      _user.save(function () {
+        _user2.save(function (err) {
           should.exist(err);
-          done();
+          _user.remove(function (err) {
+            should.not.exist(err);
+            done();
+          });
+          
```

Comment:
nitpick, but inconsistent space? If we want more spacing, we should have above and below.